app/controllers/users_controller.rb

```diff
@@ -13,4 +13,37 @@ def login_user
       render :login_form    
     end  
   end
+
+  def log_out
+    session.clear
+    redirect_to root_path
+  end
+
+  def new;end
+
+  def create
+    if params[:password] == params[:password_confirmation]
+      response = UsersFacade.create_user(params[:name], params[:email], params[:password])
+      ### thinking ahead for error handling and status codes ###
+
+      # if response[:status] == 201
+        # flash[:success] = "Account Created"
+        session[:user_id] = response.user_id
+        # require 'pry'; binding.pry
+        redirect_to user_path(response)
+      # else
+      #   flash[:error] = response[:error]
+      #   redirect_back(fallback_lcoation: new_user_path)
+      # end       
+    else
+      flash[:error] = "Passwords Must Match"
+      redirect_back(fallback_location: new_user_path)
+      # require 'pry'; binding.pry
+    end
+  end
```

Comment:
Do we want to just build the sad pathing before pushing this through? I am cool either way.